### PR TITLE
Add Github issue template for creating new or update modules.

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

**Comment by ruflin** [comment]
@andrewvc As you created these before, is there a way to test this before it gets merged?

**Comment by ruflin** [comment]
@jsoriano @sayden Would be good to have your feedback on this one if I missed some things. I hope moving forward we can use this for new modules.

**Comment by exekias** [comment]
cc @kaiyan-sheng for awareness

**Comment by jsoriano** [comment]
> @andrewvc As you created these before, is there a way to test this before it gets merged?

There is an editor where templates can be previewed here: https://github.com/elastic/beats/issues/templates/edit
Not sure if it can be used for already open PRs...

File: .github/ISSUE_TEMPLATE/module-checklist.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+name: New Module / Dataset
+about: "Meta issue to track the creation, updating of a new module or dataset."
+
+---
+
+# Metricbeat Module / Dataset release checklist
+
+This checklist is intended for Devs which create or update a module to make sure modules are consistent.
+
+The idea of having this in Markdown is that it can be copied 1-1 to a Github issue and be used as meta issue.
```
Comment (jsoriano) [inline .github/ISSUE_TEMPLATE/module-checklist.md]: No need for this line now? :slightly_smiling_face:

Comment (ruflin) [reply .github/ISSUE_TEMPLATE/module-checklist.md]: agree, removed

**Pull request merged**